docker-compose.dev-windows.yaml

```diff
@@ -0,0 +1,65 @@
+
+version: '3.8'
+
+networks:
+  app_network:
+    driver: bridge
+
+services:
+  backend:
+    build: .
+    command: ["/bin/bash", "-c", "chmod +x /app/entrypoint.sh && /app/entrypoint.sh"]
```

Comment:
why do you have duplicate `chmod` command here in and in `RUN 42 lines in Dockerfile`

Reply:
The file `entrypoint.sh` has a permission and format problem, thus this will set the executable permissions manually before running the script. First, I make sure the file has permissions. After `dos2unix` makes the change, the command is run again to make sure the permissions weren't overridden.

Reply:
I'll update the script to ensure just one `chmod` is ran.